### DISCUSSION TITLE
How to disable ClientLog endpoint

### DISCUSSION THREAD

**Discussion by xMinerv** [jellyfin/jellyfin#9939] [General]
Hi there. I don't want my jellyfin clients to be able to post to the ClientLog endpoint. Is there a way to completely disable it? I see that there was a discussion about "AllowClientLogUpload" (https://github.com/jellyfin/jellyfin/pull/6879), but I don't see how its used, and using --AllowClientLogUpload false produces the error "Option 'AllowClientLogUpload' is unknown." I'm using linux, if that matters.

Thank you so much!

**Comment by crobibero** [comment]
You can enable/disable this feature in system.xml
